pkg/sdk/external_tables_integration_test.go

```diff
@@ -0,0 +1,392 @@
+package sdk
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
This time golangci is right, we keep internal and external imports separated (github.com/stretchr/* should be on the bottom of import, with line separating them from context and fmt)